src/content/docs/tutorial-errors/respond-outages.mdx

```diff
@@ -0,0 +1,155 @@
+
+    </Step>
+    <Step>
+        ### Choose a service
+
+Let's see how this might look in practice. When you view the **All entities** page, you notice four services are alerting.
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={allEntitiesView}
+/>
+
+After asking yourself the questions from step one, you know that: 
+
+* The end user is struggling with purchase actions.
+* Your site should only display in-stock items.
+* Your site is displaying all products, so customers are able to purchase out-of-stock items.
+
+After performing your audit, you know that `api-gateway` is a critical dependency for your inventory. This is where you'll begin your error analysis. 
+
+    </Step>
+    <Step>
+
+### Locate what changed [#source] 
+
+You have your entry point into your system, so now you can look into the errors affecting your app. From the `api-gateway` summary page, click the **Errors** tab under **Triage**. Your errors page filters your data to an errors-only view. 
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={apmErrorsInboxPage}
+/>
+
+`api-gateway` has at least six error groups reporting with anywhere from a dozen to thousands of occurrences in your app. Don't make any assumptions about what error is critical. Instead, start with the time series on the far right of the waterfall view. 
+
+At first this seems to lack granularity, but your time series gives you enough information to point to what changed over time. We'll break this down:
+
+* Based on number of occurrences alone, your first instinct might tell you to start with `ActivemModel:::ValidationError` as it has 4,000 occurrences. If you look at the time series, though, its peaks and troughs are relatively consistent. This could be an expected error, but let's look at the other five. 
+* The `Net::OpenTimeOut` error group has a similar pattern, and it actually makes up four of the six reporting groups. Across each error group, you can see consistent peaks and troughs that extend before the incident. With the same name and similar patterns, we can infer this is an expected error as well.
+* Our last option is `JsonapiClient:::Notfound`. Like `ActivemModel:::ValidationError`, it has a distinct shape and is consistently reporting. While it doesn’t have many occurrences, the timeseries is anomalous enough that it might be worth digging a bit deeper. 
+    </Step>
+    <Step>
+    ### Adjust the timeseries [#timeseries]
+
+To be certain, adjust the time parameter to show patterns from the last 12 hours:
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={apmErrorsAnomaly}
+/>
+
+With the adjustment, you see that `ActivemModel:::ValidationError`has an unchanging pattern of peaks and troughs, but your  `JsonapiClient:::Notfound` has a dramatic change in behavior. This is a good ground zero. 
```

Comment:
`ground zero` is probably too metaphorical

Reply:
`has a dramatic change in behavior` perhaps more human and specific language, like `increased dramatically in the last hour`